src/lib/components/chat-window/chat-window.component.ts

```diff
@@ -72,7 +74,12 @@ export class ChatWindowComponent implements OnInit, OnDestroy {
     }
 
     async uploadFile(file: File) {
-        const url = await this.httpFileUploadPlugin.upload(file);
```

Comment:
It should be possible to handle it over provide. HttpFileUploadPlugin would also implement FileUploadHandler.

The project provides for FILE_UPLOAD_HANDLER_TOKEN HttpFileUploadPlugin

Reply:
Done with commit 755e6a0.